specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1955 @@
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/LocationList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{ascLocation}": {
+            "get": {
+                "tags": ["Locations"],
+                "description": "Details of a specific location",
+                "operationId": "Locations_Get",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }, {
+                    "$ref": "#/parameters/AscLocation"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Location"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/tasks": {
+            "get": {
+                "tags": ["Tasks"],
+                "description": "Recommended tasks that will help improve the security of the subscription proactively",
+                "operationId": "TasksSubscription_List",
+                "parameters": [{
+                    "$ref": "#/parameters/ApiVersion"
+                }, {
+                    "$ref": "#/parameters/SubscriptionId"
+                }, {
+                    "$ref": "#/parameters/ODataFilter"
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/TaskList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{ascLocation}/tasks": {

```

Comment:
For all your APIs, do you need to provide the resources by location?

Reply:
only some of them
this swagger describe the types correctly (with or without locations)